pkg/awsutils/awssession/session.go

```diff
@@ -89,6 +91,7 @@ func New() *session.Session {
 
 // injectUserAgent will inject app specific user-agent into awsSDK
 func injectUserAgent(handlers *request.Handlers) {
+	version = utils.GetEnv(envVpcCniVersion, "v1.15.0")
```

Comment:
Let's remove `version` as a global variable and make the default be the empty string, which matches the current behavior.